install.sh

```diff
@@ -177,3 +177,5 @@ fi
 # Installation Complete
 echo
 echo [~] Installation Complete....
+echo [~] Restart the terminal to affect changes.
```

Comment:
I think `Restart the terminal for the changes to take effect.` seems more correct. What do you think?